addons/wowl/static/tests/helpers/mocks.ts

```diff
@@ -80,29 +79,33 @@ export function makeFakeMenusService(menuData?: MenuData): Service<MenuService>
       return menusService;
     },
   };
+}*/
+
+function buildMockRPC(mockRPCs?: MockRPC | MockRPC[]) {
+  return async (...args: Parameters<RPC>) => {
+    let res;
+    if (mockRPCs) {
+      if (Array.isArray(mockRPCs)) {
```

Comment:
do you have a usecase where mockRPCs is an array?

Reply:
not really, The previous implementation had a global array to make sure other modules could push in it and cover default routes. Not sure if we will need it later on....